src/Compilers/Core/Portable/Operations/ControlFlowRegionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis
```

Comment:
I will move this file to Workspaces layer, this internal helper already exists in ControlFlowGraphBuilder in the compiler layer.